lib/src/Pos/Network/Block/Types.hs

```diff
@@ -54,9 +54,9 @@ data MsgGetHeaders = MsgGetHeaders
     } deriving (Generic, Show, Eq)
 
 instance Buildable MsgGetHeaders where
-    build (MsgGetHeaders mghFrom mghTo) =
+    build (MsgGetHeaders mghFrom' mghTo') =
```

Comment:
;Name clash with `mghFrom` and `mghTo`: https://github.com/input-output-hk/cardano-sl/blob/22bebb9c2754b4c9673f03e27127396d51da3ffb/lib/src/Pos/Network/Block/Types.hs#L52